fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraTimeMap.java

```diff
@@ -22,9 +22,5 @@
  * @since Oct. 04, 2017
  */
 public interface FedoraTimeMap {
```

Comment:
Sinde FedoraTimeMapImpl extends FedoraResourceImpl, FedoraTimeMap should extend FedoraResource.